src/Symfony/Component/DependencyInjection/ContainerInterface.php

```diff
@@ -21,7 +22,7 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  */
-interface ContainerInterface
+interface ContainerInterface extends PsrContainerInterface
```

Comment:
I'm not talking about reducing the feature set, I'm saying, if we use the container in our internal class A, and we only call `get` and `has` on it, then we should change the type hinting. Not sure how this plays with extension, let me check